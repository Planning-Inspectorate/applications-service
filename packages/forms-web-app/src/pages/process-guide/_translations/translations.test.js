const enProcessGuideTranslations = require('./en.json');
const cyProcessGuideTranslations = require('./cy.json');

describe('pages/process-guide/_translations', () => {
	it('should return the correct English translations', () => {
		expect(enProcessGuideTranslations).toMatchSnapshot();
	});

	it('should return the correct Welsh translations', () => {
		expect(cyProcessGuideTranslations).toMatchSnapshot();
	});

	it('should return the same keys for English and Welsh translations', () => {
		expect(enProcessGuideTranslations).toHaveSameKeys(cyProcessGuideTranslations);
	});
});
