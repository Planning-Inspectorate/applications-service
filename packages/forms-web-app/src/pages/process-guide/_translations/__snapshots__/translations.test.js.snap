// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages/process-guide/_translations should return the correct English translations 1`] = `
{
  "acceptance": {
    "heading1": "Acceptance",
    "linkText": "How the acceptance stage works and what happens next.",
    "paragraph1": "This is when the applicant sends us their application documents. We check if we can accept the application for examination. We have 28 days to make this decision.",
  },
  "common": {
    "processGuide": "Process guide",
  },
  "decision": {
    "heading1": "Decision",
    "linkText": "Who makes the final decision.",
    "paragraph1": "The decision stage is when the relevant Secretary of State then reviews the report and makes the final decision. They have 3 months to make a decision.",
  },
  "examination": {
    "heading1": "Examination",
    "linkText": "What happens at the examination stage?",
    "paragraph1": "The Examining Authority will ask questions about the proposed development. The applicant and anyone who has registered to have their say can get involved and submit comments at each deadline in the timetable. You can also attend hearings that may take place. This stage takes up to 6 months.",
  },
  "index": {
    "heading1": "The process for Nationally Significant Infrastructure Projects (NSIPs)",
  },
  "materialChange": {
    "applicationPublished": {
      "heading1": "Application published",
      "paragraph1": "The applicant notifies anyone who has been consulted and publicises the fact that they have made an application. This must include how and when parties can register to get involved. The time period for registering is set by the applicant but must be no less than 28 days.",
      "paragraph2": "Anyone who wants to have their say needs to register at this stage.",
      "paragraph3": "After the registration period has closed, the Secretary of State will review the application. They will decide whether the application needs to be examined in detail.",
      "paragraph4": "If the Secretary of State believes an examination is not required, they'll ask for more information before they make a decision.",
      "paragraph5": "If an examination is required, they'll appoint an Examining Body to carry this out.",
    },
    "applicationReceived": {
      "heading1": "Application received",
      "paragraph1": "We will publish the application when we receive it.",
    },
    "decision": {
      "heading1": "Decision",
      "paragraph1": "The relevant Secretary of State makes the final decision.",
      "paragraph2": "They'll consider the relevant representations received.",
      "paragraph3": "If an examination was required, the decision will be based on the recommendation report.",
      "paragraph4": "The Secretary of State has 2 months to make a decision.",
    },
    "examination": {
      "heading1": "Examination",
      "paragraph1": "The Examining Body asks questions about the proposed changes to the Development Consent Order.",
      "paragraph2": "The applicant and anyone who has registered to have their say can get involved and submit comments at each deadline in the timetable. You can also attend hearings that may take place.",
      "paragraph3": "This stage takes up to 4 months.",
    },
    "postDecision": {
      "heading1": "What happens after the decision is made",
      "paragraph1": "Once the Secretary of State has made a decision, challenges can be made to the High Court. All procedures must be followed when making a challenge. The High Court will decide if there are grounds for a judicial review.",
      "paragraph2": "This must happen within 6 weeks.",
    },
    "preApplication": {
      "heading1": "Pre-application",
      "paragraph1": "The applicant plans their application to make a change to the Development Consent Order (DCO).",
      "paragraph2": "They carry out consultations and publicise the proposed application. This includes gathering information about how the changes to the project could affect the local environment.",
      "paragraph3": "It is important to get involved at this stage to influence the application before the applicant sends it to the Planning Inspectorate.",
    },
    "recommendation": {
      "heading1": "Recommendation",
      "paragraph1": "The Examining Body writes its recommendation report. This must be completed and sent to the relevant Secretary of State within 2 months of the end of examination stage.",
    },
  },
  "postDecision": {
    "heading1": "What happens after the decision is made",
    "linkText": "What you can do after the decision has been made.",
    "paragraph1": "Once the Secretary of State has made a decision, challenges can be made to the High Court. All procedures must be followed when making a challenge. The High Court will decide if there are grounds for a judicial review.",
    "paragraph2": "This must happen within 6 weeks.",
  },
  "preApplication": {
    "heading1": "Pre-application",
    "linkText": "Find out what you can do at this stage and check our detailed guides.",
    "paragraph1": "This is where the applicant starts to create their application. The applicant is required to consult with people and organisations in the area. They must also create detailed documents about the impact the project could have on the environment.",
    "paragraph2": "It is important to get involved at this stage to influence the application before the applicant sends it to the Planning Inspectorate.",
  },
  "preExamination": {
    "heading1": "Pre-examination",
    "linkText": "What happens during the pre-examination stage.",
    "paragraph1": "The Examining Authority is appointed and is made up of one or more inspectors. Anyone who wants to have their say needs to register at this stage.",
    "paragraph2": "The applicant must publish that the application has been accepted by us. They include when and how parties can register to get involved. The time period for registering is set by the applicant but must be no less than 28 days.",
    "paragraph3": "The pre-examination stage usually takes about 3 months.",
  },
  "recommendation": {
    "heading1": "Recommendation",
    "linkText": "Making a recommendation.",
    "paragraph1": "The Examining Authority writes its recommendation report. This must be completed and sent to the relevant Secretary of State within 3 months of the end of examination stage.",
  },
}
`;

exports[`pages/process-guide/_translations should return the correct Welsh translations 1`] = `
{
  "acceptance": {
    "heading1": "Derbyn",
    "linkText": "Sut mae'r cam derbyn yn gweithio a beth fydd yn digwydd nesaf.",
    "paragraph1": "Dyma'r adeg pan fydd yr ymgeisydd yn anfon ei ddogfennau cais atom. Byddwn yn gwirio a allwn dderbyn y cais i'w archwilio. Mae gennym 28 niwrnod i wneud y penderfyniad hwn.",
  },
  "common": {
    "processGuide": "Canllaw proses",
  },
  "decision": {
    "heading1": "Penderfyniad",
    "linkText": "Pwy sy'n gwneud y penderfyniad terfynol.",
    "paragraph1": "Y cam penderfynu yw'r adeg pan fydd yr Ysgrifennydd Gwladol perthnasol yn adolygu'r adroddiad ac yn gwneud y penderfyniad terfynol. Mae ganddo 3 mis i wneud penderfyniad.",
  },
  "examination": {
    "heading1": "Archwiliad",
    "linkText": "Beth sy'n digwydd yn ystod y cam archwilio?",
    "paragraph1": "Bydd yr Awdurdod Archwilio'n gofyn cwestiynau am y datblygiad arfaethedig. Gall yr ymgeisydd ac unrhyw un sydd wedi cofrestru i leisio'i farn gymryd rhan a chyflwyno sylwadau erbyn pob terfyn amser yn yr amserlen. Gallwch hefyd fynychu gwrandawiadau a allai gael eu cynnal. Mae'r cam hwn yn cymryd hyd at 6 mis.",
  },
  "index": {
    "heading1": "Y broses ar gyfer Prosiectau Seilwaith o Arwyddocâd Cenedlaethol (NSIPau)",
  },
  "materialChange": {
    "applicationPublished": {
      "heading1": "Cais wedi'i gyhoeddi",
      "paragraph1": "Bydd yr ymgeisydd yn rhoi gwybod i unrhyw un yr ymgynghorwyd ag ef ac yn rhoi cyhoeddusrwydd i'r ffaith ei fod wedi gwneud cais. Mae'n rhaid i hyn gynnwys sut a phryd y gall partïon gofrestru i gymryd rhan. Mae'r cyfnod ar gyfer cofrestru yn cael ei osod gan yr ymgeisydd ond mae'n rhaid iddo beidio â bod yn llai na 28 niwrnod.",
      "paragraph2": "Bydd angen i unrhyw un sydd eisiau lleisio'i farn gofrestru yn ystod y cam hwn",
      "paragraph3": "Ar ôl i'r cyfnod cofrestru gau, bydd yr Ysgrifennydd Gwladol yn adolygu'r cais. Bydd yn penderfynu a oes angen i'r cais gael ei archwilio'n fanwl.",
      "paragraph4": "Os bydd yr Ysgrifennydd Gwladol yn credu nad oes angen archwiliad, bydd yn gofyn am fwy o wybodaeth cyn gwneud penderfyniad.",
      "paragraph5": "Os bydd angen archwiliad, bydd yn penodi Corff Archwilio i'w gynnal",
    },
    "applicationReceived": {
      "heading1": "Cais wedi'i dderbyn",
      "paragraph1": "Byddwn yn cyhoeddi'r cais pan fyddwn yn ei dderbyn.",
    },
    "decision": {
      "heading1": "Penderfyniad",
      "paragraph1": "Bydd yr Ysgrifennydd Gwladol perthnasol yn gwneud y penderfyniad terfynol.",
      "paragraph2": "Bydd yn ystyried y sylwadau perthnasol a dderbyniwyd.",
      "paragraph3": "Os oedd angen archwiliad, bydd y penderfyniad wedi'i seilio ar yr adroddiad argymhelliad",
      "paragraph4": "Bydd gan yr Ysgrifennydd Gwladol 2 fis i wneud penderfyniad.",
    },
    "examination": {
      "heading1": "Archwiliad",
      "paragraph1": "Bydd y Corff Archwilio'n gofyn cwestiynau am y newidiadau arfaethedig i'r Gorchymyn Caniatâd Datblygu",
      "paragraph2": "Gall yr ymgeisydd ac unrhyw un sydd wedi cofrestru i leisio'i farn gymryd rhan a chyflwyno sylwadau erbyn pob dyddiad cau yn yr amserlen. Gallwch fynychu gwrandawiadau a allai gael eu cynnal hefyd.",
      "paragraph3": "Bydd y cam hwn yn cymryd hyd at 4 mis.",
    },
    "postDecision": {
      "heading1": "Beth fydd yn digwydd ar ôl i'r penderfyniad gael ei wneud",
      "paragraph1": "Pan fydd yr Ysgrifennydd Gwladol wedi gwneud penderfyniad, gellir cyflwyno heriau i'r Uchel Lys. Mae'n rhaid dilyn yr holl weithdrefnau wrth gyflwyno her. Bydd yr Uchel Lys yn penderfynu a oes sail ar gyfer adolygiad barnwrol.",
      "paragraph2": "Mae'n rhaid i hyn ddigwydd o fewn 6 wythnos.",
    },
    "preApplication": {
      "heading1": "Cyn-ymgeisio",
      "paragraph1": "Bydd yr ymgeisydd yn cynllunio ei gais i wneud newid i'r Gorchymyn Caniatâd Datblygu (DCO).",
      "paragraph2": "Bydd yn cynnal ymgynghoriadau ac yn rhoi cyhoeddusrwydd i'r cais arfaethedig. Mae hyn yn cynnwys casglu gwybodaeth am sut gallai'r newidiadau i'r prosiect effeithio ar yr amgylchedd lleol.",
      "paragraph3": "Mae'n bwysig cymryd rhan ar y cam hwn i ddylanwadu ar y cais cyn i'r ymgeisydd ei anfon at yr Arolygiaeth Gynllunio.",
    },
    "recommendation": {
      "heading1": "Argymhelliad",
      "paragraph1": "Bydd y Corff Archwilio'n ysgrifennu ei adroddiad argymhelliad. Mae'n rhaid i hwn gael ei gwblhau a'i anfon at yr Ysgrifennydd Gwladol perthnasol o fewn 2 fis o ddiwedd y cam archwilio.",
    },
  },
  "postDecision": {
    "heading1": "Beth sy'n digwydd ar ôl i'r penderfyniad gael ei wneud",
    "linkText": "Yr hyn y gallwch ei wneud ar ôl i'r penderfyniad gael ei wneud.",
    "paragraph1": "Pan fydd yr Ysgrifennydd Gwladol wedi gwneud penderfyniad, gellir cyflwyno heriau i'r Uchel Lys. Mae'n rhaid dilyn yr holl weithdrefnau wrth gyflwyno her. Bydd yr Uchel Lys yn penderfynu a oes sail ar gyfer adolygiad barnwrol.",
    "paragraph2": "Mae'n rhaid i hyn ddigwydd o fewn 6 wythnos.",
  },
  "preApplication": {
    "heading1": "Cyn-ymgeisio",
    "linkText": "Dysgwch beth allwch chi ei wneud yn ystod y cam hwn a gwiriwch ein canllawiau manwl.",
    "paragraph1": "Dyma'r adeg pan fydd yr ymgeisydd yn dechrau creu ei gais. Mae'n ofynnol i'r ymgeisydd ymgynghori â phobl a sefydliadau yn yr ardal. Mae'n rhaid iddo hefyd greu dogfennau manwl ynglŷn â'r effaith y gallai'r prosiect ei chael ar yr amgylchedd.",
    "paragraph2": "Mae'n bwysig cymryd rhan yn ystod y cam hwn er mwyn dylanwadu ar y cais cyn i'r ymgeisydd ei anfon at yr Arolygiaeth Gynllunio.",
  },
  "preExamination": {
    "heading1": "Cyn-archwilio",
    "linkText": "Beth sy'n digwydd yn ystod y cam cyn-archwilio.",
    "paragraph1": "Penodir Awdurdod Archwilio sy'n cynnwys un neu fwy o arolygwyr. Mae'n rhaid i unrhyw un sydd eisiau lleisio'i farn allu cofrestru yn ystod y cam hwn.",
    "paragraph2": "Mae'n rhaid i'r ymgeisydd gyhoeddi bod y cais wedi cael ei dderbyn gennym. Bydd yn cynnwys pryd a sut y gall partïon gofrestru i gymryd rhan. Mae'r cyfnod ar gyfer cofrestru'n cael ei osod gan yr ymgeisydd, ond mae'n rhaid iddo beidio â bod yn llai na 28 niwrnod.",
    "paragraph3": "Mae'r cam cyn-archwilio'n cymryd oddeutu 3 mis fel arfer.",
  },
  "recommendation": {
    "heading1": "Argymhelliad",
    "linkText": "Gwneud argymhelliad.",
    "paragraph1": "Mae'r Awdurdod Archwilio'n ysgrifennu ei adroddiad argymhelliad. Mae'n rhaid i hwn gael ei gwblhau a'i anfon at yr Ysgrifennydd Gwladol perthnasol o fewn 3 mis o ddiwedd y cam archwilio.",
  },
}
`;
